junit-tests/src/test/java/org/testmonkeys/sut/demoqa/playground/RadioComponentTest.java

```diff
@@ -0,0 +1,91 @@
+package org.testmonkeys.sut.demoqa.playground;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.testmonkeys.koshmar.core.browser.Browser;
+import org.testmonkeys.koshmar.core.browser.DriverFactory;
+import org.testmonkeys.koshmar.core.factory.PageFactory;
+import org.testmonkeys.koshmar.core.factory.PageScanner;
+import org.testmonkeys.sut.demoqa.RegistrationPage;
+
+import java.util.NoSuchElementException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.hamcrest.collection.IsArrayContaining.*;
+
+
+public class RadioComponentTest {
+
+    private DriverFactory df = new DriverFactory();
+    private Browser browser = new Browser(df.initDriver("chrome"));
+    private PageFactory pageFactory = new PageFactory(browser, new PageScanner("org.testmonkeys.sut"),
+            "http://demoqa.com");
+
+    RegistrationPage registrationPage = pageFactory.createPage(RegistrationPage.class);
+
+    @Rule
+    public ExpectedException expectedException=ExpectedException.none();
+
+    @Before
+    public void beforeScenario() {
+        registrationPage.open();
+    }
+
+    @After
+    public void cleanup(){
+        browser.getDriver().close();
```

Comment:
use quit instead of close, otherwise the chromedriver process is not killed and it gets stuck in processes